sigstore/_cli.py

```diff
@@ -423,11 +424,11 @@ def _verify(args: argparse.Namespace) -> None:
     for file, inputs in input_map.items():
         # Load the signing certificate
         logger.debug(f"Using certificate from: {inputs['cert']}")
-        certificate = inputs["cert"].read_bytes()
+        certificate = inputs["cert"].read_bytes().rstrip()
 
         # Load the signature
         logger.debug(f"Using signature from: {inputs['sig']}")
-        signature = inputs["sig"].read_bytes()
+        signature = inputs["sig"].read_bytes().rstrip()
```

Comment:
N.B.: This wasn't causing failures, but I noticed that these two inputs can sometimes have trailing newlines in them (due to being text files).